app/vue/package.json

```diff
@@ -70,17 +70,22 @@
     "url-loader": "^0.6.2",
     "util-deprecate": "^1.0.2",
     "uuid": "^3.1.0",
-    "vue": "^2.5.2",
-    "vue-hot-reload-api": "^2.2.0",
-    "vue-loader": "^12.2.1",
+    "vue-hot-reload-api": "^2.1.1",
     "vue-style-loader": "^3.0.1",
-    "vue-template-compiler": "^2.5.2",
     "webpack": "^3.6.0",
     "webpack-dev-middleware": "^1.12.0",
     "webpack-hot-middleware": "^2.20.0"
   },
   "devDependencies": {
     "babel-cli": "^6.26.0",
-    "nodemon": "^1.12.1"
+    "nodemon": "^1.12.1",
+    "vue": "^2.5.2",
+    "vue-loader": "^13.3.0",
```

Comment:
What are the breaking changes between 12 and 13?

Reply:
@Hypnosphi https://github.com/vuejs/vue-loader/releases/tag/v13.0.0

Upgrading `vue-loader` also fixes https://github.com/vuejs/vue-loader/issues/951 (supporting nested css rules). Which is something I use heavily in my .vue files and that's currently blocking me from adding several components to my storybook.

So would love to see this getting merged eventually!

Reply:
✅